src/lib/syncable_ledger/syncable_ledger.ml

```diff
@@ -670,12 +670,12 @@ end = struct
     in
     if not should_skip then (
       Option.iter t.desired_root ~f:(fun root_hash ->
-          Logger.info t.logger ~module_:__MODULE__ ~location:__LOC__
+          Logger.debug t.logger ~module_:__MODULE__ ~location:__LOC__
             ~metadata:
               [ ("old_root_hash", Root_hash.to_yojson root_hash)
               ; ("new_root_hash", Root_hash.to_yojson h) ]
-            "new_goal: changing target from $old_root_hash to $new_root_hash"
-      ) ;
+            "New_goal in sync ledger changing target from $old_root_hash to \
```

Comment:
I approve of the move to `debug` but the `~module_` should convey the `sync ledger` aspect well enough to not need it in the message, imo?